utils/cp_utils.py

```diff
@@ -0,0 +1,71 @@
+"""
+This collection of functions runs CellProfiler and will rename the .sqlite outputed to any specified name if running an analysis pipeline.
+"""
+
+import os
+import pathlib
+
+
+def rename_sqlite_file(sqlite_dir_path: pathlib.Path, name: str):
+    """Rename the .sqlite file to be {name}.sqlite as to differentiate between the files.
+
+    Args:
+        sqlite_dir_path (pathlib.Path): path to CellProfiler_output directory
+        name (str): new name for the SQLite file
+    """
+    try:
+        # CellProfiler requires a name to be set in to pipeline, so regardless of plate or method, all sqlite files name are hardcoded
+        sqlite_file_path = pathlib.Path(f"{sqlite_dir_path}/NF1_data.sqlite")
+
+        new_file_name = str(sqlite_file_path).replace(
+            sqlite_file_path.name, f"{name}.sqlite"
+        )
+
+        # change the file name in the directory
+        pathlib.Path(sqlite_file_path).rename(pathlib.Path(new_file_name))
+        print(f"The file is renamed to {pathlib.Path(new_file_name).name}!")
+
+    except FileNotFoundError as e:
```

Comment:
This is up for discussion. I'd recommend placing exceptions documentations within the function doc string. It provides a general explanation why these exceptions are raised. 

https://stackoverflow.com/questions/51994464/documenting-exceptions-that-can-happen-in-other-functions-in-python-docstrings

Reply:
I like this! I am going to add this exception to the doc string now!